torchaudio/csrc/CMakeLists.txt

```diff
@@ -126,6 +126,51 @@ define_library(
   "${LIBTORCHAUDIO_COMPILE_DEFINITIONS}"
   )
 
+
+################################################################################
+# libtorchaudio_decoder.so
+################################################################################
+if (BUILD_CTC_DECODER)
+  set(
+    LIBTORCHAUDIO_DECODER_SOURCES
+    decoder/src/decoder/Decoder.h
```

Comment:
we should be able to remove the `.h` header files from this list

Reply:
Thanks for pointing. Let me follow-up on this in the subsequent PR so that I can land this without alerting the fbcode's change-after-approve system during the weekend.